--- drakconf-12.19.2/control-center.orig	2013-10-23 23:04:05.800922278 +0100
+++ drakconf-12.19.2/control-center	2013-11-18 11:24:57.955036732 +0000
@@ -665,9 +665,9 @@
                      ),
                   [ N("/_Help"), undef, undef, undef, '<Branch>' ],
                   [ N("/_Help") . N("/_Help"), undef, \&run_help, undef, '<Item>', N("Help") ],
-                  [ N("/_Help") . N("/_Release notes"), undef,  sub { run_browser('Notes') }, undef, '<Item>', N("Help") ],
-                  [ N("/_Help") . N("/What's _New?"), undef,  sub { run_browser("What's New?") }, undef, '<Item>', N("Help") ],
-                  [ N("/_Help") . N("/_Errata"), undef,  sub { run_browser('Errata') }, undef, '<Item>', N("Help") ],
+                  [ N("/_Help") . N("/_Release notes"), undef,  sub { run_browser('Release_Notes') }, undef, '<Item>', N("Help") ],
+                  [ N("/_Help") . N("/What's _New?"), undef,  sub { run_browser("New") }, undef, '<Item>', N("Help") ],
+                  [ N("/_Help") . N("/_Errata"), undef,  sub { run_browser('Release_Notes#Errata') }, undef, '<Item>', N("Help") ],
                   [ N("/_Help") . N("/_Report Bug"), undef, sub {
                         run_program::raw({ detach => 1, as_user => 1 }, 'drakbug', '--report', (split(/\s/, $pending_app))[0] || 'drakconf');
                       }, undef, '<Item>' ],
@@ -681,7 +681,7 @@
 
 sub run_browser {
     my ($url) = @_;
-    $url = "http://wiki.mandriva.com/en/$release->{version}_$url" if $url !~ m!^http://!;
+    $url = "http://wiki.openmandriva.org/en/$release->{version}/$url" if $url !~ m!^http://!;
     run_program::raw({ detach => 1, as_user => 1 }, '/usr/bin/www-browser', $url);
 }
 
