--- drakconf-12.19.3/lib/MDV/Control_Center.pm.omv~	2014-04-04 10:05:34.599063517 +0200
+++ drakconf-12.19.3/lib/MDV/Control_Center.pm	2014-04-04 10:05:44.628063107 +0200
@@ -84,8 +84,8 @@ our %programs =
 
    "Boot Config" =>
    {
-    binary => "$sbindir/drakboot --boot",
-    embedded => 1,
+    binary => "$bindir/kcmshell4 grub2",
+    embedded => -1,
     description => N("Set up boot system"),
     long_description => N("Set up how the system boots"),
     icon => 'drakboot-mdk',
